src/ixdat/measurements.py

```diff
@@ -269,22 +286,203 @@ def data_cols(self):
 
-    def plot(self, plotter=None, *args, **kwargs):
-        """Plot the measurement using its plotter (see its Plotter for details)"""
-        if plotter:
-            return plotter.plot_measurement(self, *args, **kwargs)
-        if not self.plotter:
+    @property
+    def plotter(self):
+        """The default plotter for Measurement is ValuePlotter."""
+        if not self._plotter:
             from .plotters import ValuePlotter
 
-            self.plotter = ValuePlotter(measurement=self)
-        return self.plotter.plot(*args, **kwargs)
+            self._plotter = ValuePlotter(measurement=self)
+        # self.plot_measurement.__doc__ = self._plotter.plot_measurement.__doc__
+        # self.plot.__doc__ = self._plotter.plot_measurement.__doc__
+        # FIXME: Help! plot_measurement() needs to be wrapped with the plotter's
+        # plot_measu
+        return self._plotter
 
-    def export(self, exporter=None, *args, **kwargs):
+    @property
+    def exporter(self):
+        """The default exporter for Measurement is CSVExporter."""
+        if not self._exporter:
+            self._exporter = CSVExporter(measurement=self)
+        return self._exporter
+
+    def export(self, *args, exporter=None, **kwargs):
         """Export the measurement using its exporter (see its Exporter for details)"""
         if exporter:
             return exporter.export_measurement(self, *args, **kwargs)
         return self.exporter.export(*args, **kwargs)
 
+    def get_original_m_id_of_series(self, series):
+        """Return the id(s) of component measurements to which `series` belongs."""
+        m_id_list = []
+        for m in self.component_measurements:
+            if series.id in m.s_ids:
+                m_id_list.append(m.id)
+        if len(m_id_list) == 1:
+            return m_id_list[0]
+        return m_id_list
+
+    def cut(self, tspan):
+        """Return a new measurement with the data in the given time interval
+
+        Args:
+            tspan (iter of float): The time interval to use, relative to self.tstamp
+                tspan[0] is the start time of the interval, and tspan[-1] is the end
+                time of the interval. Using tspan[-1] means you can directly use a
+                long time vector that you have at hand to describe the time interval
+                you're looking for.
+        """
+        new_series_list = []
+        obj_as_dict = self.as_dict()
+        time_cutting_stuff = {}  # {tseries_id: (mask, new_tseries)}
+        for series in self.series_list:
+            try:
+                tseries = series.tseries
+                if tseries is None:
+                    raise AttributeError
+            except AttributeError:  # series independent of time are uneffected by cut
+                new_series_list.append(series)
+            else:
+                t_id = (tseries.id, tseries.backend_name)
+                # FIXME: Beautiful, met my first id clash here. Local memory and loaded
```

Comment:
Would a solution to this be using large random numbers as identifiers?

Reply:
Yeah, Kenneth has made the same point. I first resonded that I want to see it become necessary first (see https://github.com/ixdat/ixdat/pull/1#discussion_r546400226 and https://github.com/ixdat/ixdat/pull/1#issuecomment-754203224). And here it is. But now I have a new argument which is using [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier) instead lose the readability of the id's, which I find very useful when playing with it and the `DirBackend` (see slides). 
It could be good to have both the small-integer id and a UUID. 
But this (id, backend_name) could also work for a while. I think in any case it can wait til we have an actual SQL backend to test things against. There's TODO's in the code so we won#t forget ;)